src/internal/kopia/upload_test.go

```diff
@@ -21,6 +24,107 @@ import (
 	"github.com/alcionai/corso/src/pkg/path"
 )
 
+func makePath(t *testing.T, elements []string) path.Path {
+	p, err := path.FromDataLayerPath(stdpath.Join(elements...), false)
+	require.NoError(t, err)
+
+	return p
+}
+
+type expectedNode struct {
+	name     string
+	children []*expectedNode
+	data     []byte
+}
+
+// Currently only works for files that Corso has serialized as it expects a
+// version specifier at the start of the file.
+//
+//revive:disable:context-as-argument
+func expectFileData(
+	t *testing.T,
+	ctx context.Context,
+	expected []byte,
+	f fs.StreamingFile,
+) {
+	//revive:enable:context-as-argument
+	t.Helper()
+
+	if expected == nil {
```

Comment:
actually since this function assumes the version specifier is in there it's fine to do a check on length because there should be no files of size 0 (the minimum file size is the length of the version)